### PR TITLE
Tremor assessment

### PR DESCRIPTION
**Pull request by jakekrog**
This PR adds support for the tremor assessment module in ResearchKit.

@Erin-Mounts:
Regarding the issue with result calculation you raised in the other PR, I have to admit I'm a bit out of my depth; I looked into the existing score calculation code over the weekend, but I don't know if any of the existing code for processing accelerometer / motion data is applicable here.

I added more logic to the tremor view controller to help with creating the result summary, but PDScores needs to be updated to support tremor assessment data. If you have any suggestions, please let me know - I'm happy to help in any way that I can.

### CODE REVIEW

**Comment by Erin-Mounts** [comment]
Turns out, according to Andrew, since the module is new, we don't really know how to calculate a meaningful score, so for this version of the app we should just not have a dashboard item for the tremor module at all.

**Comment by jakekrog** [comment]
Thanks, that makes sense. I'll update the PR with the tremor assessment removed from the dashboard.

**Comment by syoung-smallwisdom** [comment]
Looks good

File: mPowerSDK/Dashboard/APHDashboardViewController.m [hunk 1]
```diff
@@ -420,7 +427,7 @@ - (void)prepareData
                     item.editable = YES;
                     item.tintColor = [UIColor colorForTaskId:item.taskId];
                     
-                    item.info = NSLocalizedStringWithDefaultValue(@"APH_DASHBOARD_MEMORY_INFO", nil, APHLocaleBundle(), @"This plot shows the score you received each day for the Memory Game. The length and position of each vertical bar represents the range of scores for a given day. Any differences in length or position over time reflect variations and trends in your score, which may reflect variations and trends in your symptoms.", @"Dashboard tooltip item info text for Memory in Parkinson");
+                    item.info = NSLocalizedStringWithDefaultValue(@"APH_DASHBOARD_MEMORY_INFO", nil, APHLocaleBundle(), @"== The length and position of each vertical bar represents the range of scores for a given day. Any differences in length or position over time reflect variations and trends in your score, which may reflect variations and trends in your symptoms.", @"Dashboard tooltip item info text for Memory in Parkinson");
```
Comment (Erin-Mounts) [inline mPowerSDK/Dashboard/APHDashboardViewController.m:430]: Typo--inadvertent edit?

File: mPowerSDKTests/APHActivityManagerTests.m [hunk 2]
```diff
@@ -121,6 +121,26 @@ - (void)testCreateOrderedTask_Walking_English
...
+    XCTAssertNotNil([task stepWithIdentifier:@"tremor.handInLap"]);
+    XCTAssertNotNil([task stepWithIdentifier:@"tremor.handAtShoulderLength"]);
+    XCTAssertNotNil([task stepWithIdentifier:@"tremor.handAtShoulderLengthWithElbowBent"]);
+    XCTAssertNotNil([task stepWithIdentifier:@"tremor.handToNose"]);
+    XCTAssertNotNil([task stepWithIdentifier:@"tremor.handQueenWave"]);
+    
+    // Check that the final step uses the expected language
+    ORKStep *finalStep = task.steps.lastObject;
+    XCTAssertEqualObjects(finalStep.title, @"Thank You!");
+    XCTAssertEqualObjects(finalStep.text, @"The results of this activity can be viewed on the dashboard");
```
Comment (Erin-Mounts) [inline mPowerSDKTests/APHActivityManagerTests.m]: Actually, no they can't... ;)

**Comment by Erin-Mounts** [comment]
Just waiting for word that the typo in APHDashboardViewController.m has been fixed, and will accept and merge this.

**Comment by Erin-Mounts** [comment]
Pulled locally and fixed merge (and typo). Will submit a new pull request with the updates shortly. Thanks!

**Pull request closed without merging**